dokuwiki-syntax/src/test/resources/dokuwiki10/specific/imagelink/imagelink1.test

```diff
@@ -0,0 +1,8 @@
+.#-----------------------------------------------------
+.input|dokuwiki/1.0
+.#-----------------------------------------------------
+[[http://php.net|{{wiki:dokuwiki-128.png}}]]
+.#-----------------------------------------------------
+.expect|xwiki/2.1
+.#-----------------------------------------------------
+[[[[image:dokuwiki-128.png]]>>http://php.net]]
```

Comment:
You might want to optimize a bit and set "freestanding" event parameter to true for compatible images (not white space and no parameters mainly) which will remove the [[ ]] around it.